modules/articles/client/config/articles.client.menus.js

```diff
@@ -1,8 +1,17 @@
-'use strict';
+/**
```

Comment:
@rhutchison as long as the file names are good, this is just extra. Let's try to keep it as clean as possible.